.github/workflows/check-linked-issues.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     name: Check linked issues
     steps:
-      - uses: nearform/github-action-check-linked-issues@v1
+      - uses: nearform-actions/github-action-check-linked-issues@v1
```

Comment:
it's too early for this change as, despite the warning, the action still doesn't exist in that new organization. we're working on this

Reply:
Ok, reverted the change.